packages/plugins/model-define/cycle-gan-model-define/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@pipcook/plugin-cycle-gan-model-define",
+  "version": "0.6.0",
+  "description": "",
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist"
```

Comment:
remove this dist. Otherwise CycleGAN folder will not be included in npm package

Reply:
Done.